Lib/idlelib/configdialog.py

```diff
@@ -2036,11 +2036,6 @@ def load_tab_cfg(self):
             'main', 'Indent', 'num-spaces', default=4, type='int')
         self.space_num.set(space_num)
 
-    def var_changed_space_num(self, *params):
-        "Store change to indentation size."
-        value = self.space_num.get()
-        changes.add_option('main', 'Indent', 'num-spaces', value)
```

Comment:
Sorry about that.  I was severely git-challenged yesterday.  I thought I had restored everything correctly, but I must have missed this.